keck_vnc_launcher.py

```diff
@@ -728,6 +747,56 @@ def close_firewall(self, authpass):
             self.log.error(result)
 
 
+    ##-------------------------------------------------------------------------
+    ## Check to see whether the firewall hole is already open.
+    ##-------------------------------------------------------------------------
+    def test_firewall(self):
+        ''' Return True if the sshuser firewall hole is open; otherwise
+            return False. Also return False if the test cannot be performed.
+        '''
+
+        try:
+            netcat = subprocess.check_output(['which', 'ncat'])
+        except CalledProcessError:
```

Comment:
Forgot to add that i got crash on name 'CalledProcessError' is not defined.  I changed to 'subprocess.CalledProcessError' and that worked.

Reply:
btw, the above ping issue didn't cause a crash.  It just didn't skip the firewall auth.

Reply:
Now fixed.